input/input.css

```diff
@@ -0,0 +1,55 @@
+:root{
+    --shadow-color: #b8b8b3d2;
+}
+.container{
+    padding: 1rem;
+    gap: 2rem;
+}
+.component-container{
+    padding: 2rem 4rem 2rem 2rem;
+}
+.component-details{
+    padding: 1rem;
+}
+
+.input-examples{
+    justify-content: center;
+}
+
+.input-container{
+    gap: 1rem;
+    box-shadow:  0 5px  5px var(--shadow-color), 0 6px 20px 0 var(--shadow-color);
+    padding: 2rem;
+}
+
+.input-box span{
+    color: red;
+}
+.input-box label{
+    padding: 2px;
+    font-size: 12px;
```

Comment:
looks good, if possible use rem for values greater than 10px

Reply:
Actually, 1rem was getting too big and it's said that we should prefer px over rem if a value is <1.